Makefile

```diff
@@ -58,6 +58,14 @@ stroeer/%: $(OUTPUT)
 	@echo "+ $@"
 	$(PROTOC) $(FLAGS) $(shell find $@ -iname "*.proto" -exec echo -n '"{}" ' \; | tr '\n' ' ')
 
+gateway: ## Generates grpc-gateway resources
```

Comment:
so this is an example? or do we only want to generate the gateway resources for the article_service?

Reply:
it's just a first step. We can and probably should also generate a gateway for the section service as well